app/views/record/_basic_info.html.erb

```diff
@@ -47,7 +47,7 @@
         <% if author == "et al" %>
           <%= author %>
         <% else %>
-          <%= link_to(author, search_prefix + "AU+\"#{author}\"", data: {type: "Author"} ) %><% if author != @record.eds_authors.last %>;<% end %>
+          <%= link_to(author, search_prefix + "AU+\"#{author}\"", data: {type: "Author"} ) %><% if author != @record.eds_authors.last %><% end %>
```

Comment:
This removes all the colors (so if there are multiple authors, they come across as "Botsford, Frances Yelton, Andromeda" rather than "Botsford, Frances; Yelton, Andromeda"). Is there an example URL where you're seeing colons you don't expect?

Reply:
Hmm I was seeing double colons on the full record since I put them in using css and they were hardcoded here. Are you not seeing any?

Reply:
Ohhhh, OK. In that case I'm going to remove this if statement (it no longer does anything) and add a comment so people know to find semicolon-adding functionality in css.